tests/integration/utils.py

```diff
@@ -111,7 +113,8 @@ def fetch_all_requests(self, query_params=None, all_pages=True):
         request_url = f"{self._cachito_api_url}/requests"
         all_items = []
         while request_url:
-            resp = requests.get(request_url, params=query_params, timeout=15)
+            request_session = get_requests_session()
```

Comment:
It might be better to create the request session above the `while` loop. Optionally, you could just create one global session and use it for all the calls instead of creating a new one every time (that's what Cachito does).

Reply:
Done! Thank you!

Reply:
Have you pushed your changes? I don't see them :D

Reply:
Updated right now. Wrong brunch, sorry :D